pages/guides/drop-ins/setup-and-configuration.mdx

```diff
@@ -0,0 +1,94 @@
+---
+layout: guides
+title: "Step 1: Setup and configuration"
+guide:
+  step: 1
+meta:
+  title: Building With Drop-in Components | Dwolla API Documentation
+  description: Dwolla’s Drop-in Components are low-code solutions to abstract away the complexity of integrating with a payment API and act as a shortcut to completing a payment integration.
+---
+
+# Step 1: Setup and configuration
+
+We’ll begin by setting up and configuring both the Dwolla Components library and installing a server-side SDK. This guide assumes that you have the basic structure of a web application set up and running, which includes a backend language (e.g. Node.js, Python, etc.) and a web front end (HTML, CSS, Javascript).
+
+To get up and running quickly, take a look at our [drop-ins examples repo](https://github.com/Dwolla/drop-ins-examples) which is a basic Node.js app that uses [Express](https://expressjs.com/).
+
+### Include dwolla-web.js
+
+Begin the client-side implementation by including dwolla-web.js in the ‘head’ of your HTML page. The Dwolla Components library containing all of the drop-in UI components are available directly from Dwolla’s content delivery network (CDN). As we continue to evolve the components library, it will soon be available to be installed via NPM.
+
+```html
+<head>
+  <script type="text/javascript" src="cdn.dwolla.com/2/dwolla.js"></script>
```

Comment:
I don't think `src="cdn.dwolla.com/...` will work. Pretty sure it needs to be something like `src="//cdn.dwolla.com/...`, else it will treat it as a relative path and load `https://myapp.com/cdn.dwolla.com/2/dwolla.js`.

Also, the path we have right now is `https://cdn.dwolla.com/v2/dwolla-web.js` (can be changed easily tho)

Reply:
@sausman, which one of the following should we use for this example?  
1. //cdn.dwolla.com/2/dwolla.js
or 
2. //cdn.dwolla.com/v2/dwolla-web.js